### PR TITLE
Added Readme

### LINKED ISSUE #1885: README flie for Flyod's Cycle Detection
I Have completed this

### PR DESCRIPTION
**Pull request by kmrakash**
Fixes #1885 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- I have added README file for Flyod Cycle Detection Algorithm

#### Languages Used:
- 
- markdown

#### Files Added:
- 
- README.md

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by kmrakash** [comment]
so @asha15  , i need to add algorithm and example too

**Comment by kmrakash** [comment]
@asha15  let me know if I'm going good

File: Floyd_Cycle_Detection_Algorithm/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Flyod's_Cycle_Detection
+
+The name already give idea about the algorithm , It determines if a singly linked list is circular and if it is,  It is going to return the node where the cycle begins
+It was invented by **Robert W. Floyd** in late 1960s.
+The algorithm is also popularly known as **The Tortoise and the Hare (Floyd’s Algorithm)**
```
Comment (asha15) [inline Floyd_Cycle_Detection_Algorithm/README.md:5]: can you provide some more explanation about the algo

File: Floyd_Cycle_Detection_Algorithm/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Complexity
```
Comment (asha15) [inline Floyd_Cycle_Detection_Algorithm/README.md:7]: transfer this heading to the bottom

**Comment by kmrakash** [comment]
@asha15 check if it works

**Comment by kmrakash** [comment]
Actually I'm not able to squash the commits , it shows remove conflicts .
Help @asha15

**Pull request merged**